src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/EnumDataContract.cs

```diff
@@ -364,12 +355,12 @@ internal long GetEnumValueFromString(string value)
             }
         }
 
-        public override void WriteXmlValue(XmlWriterDelegator xmlWriter, object obj, XmlObjectSerializerWriteContext context)
+        public override void WriteXmlValue(XmlWriterDelegator xmlWriter, object obj, XmlObjectSerializerWriteContext? context)
         {
             WriteEnumValue(xmlWriter, obj);
         }
 
-        public override object ReadXmlValue(XmlReaderDelegator xmlReader, XmlObjectSerializerReadContext context)
+        public override object? ReadXmlValue(XmlReaderDelegator xmlReader, XmlObjectSerializerReadContext? context)
```

Comment:
should this return non-null?

Reply:
Done. I forgot that you can change the nullability in an override.